packages/protocol/simulation/README.md

```diff
@@ -0,0 +1,17 @@
+# Simulation and Exported Data Parser and Visualization
+
+## How to use
+
+### Prerequisites
+
+For running the `simulation_data_parser.py`, you need to install the required plugins: `(python3 -m pip install -r requirements.txt)`
```

Comment:
I think we should git ingore all generated files.

Reply:
Feel free to modify this branch accordingly.